backend/routes/project.js

```diff
@@ -375,22 +377,24 @@ router.get("/info/:projectId", async (req, res) => {
     .sort({ round: -1 })
     .exec((err, proposals) => {
       Project.findById(projectId, async (err, project) => {
-        const lastProposal = proposals[proposals.length - 1];
-        const canCreateProposals = lastProposal
-          ? lastProposal.delivered.status == 2
-          : true;
         if (err) {
           res.status(400).send(err);
         }
         const airtableInfos = await getProposalRedisMultiple(
           proposals.map((x) => x.airtableRecordId),
           "."
         );
+        const canCreateProposals = !airtableInfos.some(
+          (x) =>
+            x["Proposal Standing"] === "Unreported" &&
+            x["Proposal State"] === "Funded"
+        );
```

Comment:
Proposal Standing === Unreported || In Progress || Incomplete & Inactive || In Dispute
Proposal State === Funded || Accepted

I think that covers it...?

Reply:
`canCreateProposals` to me, means we should check every condition that is required to return a simple true/false
`receivedFundingAndNotReported` is what we are writing here

These are two different things.
(2) is a subset of (1)

I'm going to create a ticket for this, and let's update w/ this for now.
Thank you!

Reply:
Ticket created => https://github.com/oceanprotocol/oceandao-proposal-portal/issues/161